docs/developing.rst

```diff
@@ -0,0 +1,58 @@
+Developing
+==========
+
+Install the requirements
+
+::
+
+    $ pip install -r requirements.txt
+
+(note that lxml, used in the tests, requires that libxslt and libxml2 be
+installed on the system)
+
+Clone the git repository:
+
+::
+
+    $ git clone https://github.com/unt-libraries/django-major-event-log
+    $ cd django-major-event-log
+
+Initialize the database
+
+::
+
+    $ python manage.py migrate
+
+Create a superuser so you can create test events
+
+::
+
+    $ python manage.py createsuperuser
+
+Start the development server
+
+::
+
+    $ python manage.py runserver
+
+(press Ctrl-c to stop the development server)
```

Comment:
We could take this out because it is standard and also, when you issue the runserver command and the server starts, it does print out a message saying that you can Ctrl-c to exit.